### DISCUSSION TITLE
Welcome to homebridge-texecom Discussions!

### DISCUSSION THREAD

**Discussion by K1LL3R234** [K1LL3R234/homebridge-texecom#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 5xPa** [comment]
Hi I have recently changed from homebridge-texecom-connect to your app. I have posted an issue regarding arming/disarming not updating on Homekit when alarm is armed/disarmed from Homekit.

I wonder if I could request you consider adding a zone_type "monitor" with in Texecom a detector can be set to monitor. It does not activated they alarm when set but does report activation. I use this function to for example warn if I forget to disarm the system before heading down stairs.

Thanks for all the hard work

**Comment by metaljay** [comment]
How does this differ from the original (non forked) version? I can only see text changes no actual programming

**Reply by K1LL3R234** [reply]
There is programming that is different in index.js check the version4.2.2 branch

**Comment by 5xPa** [comment]
I'm not a programmer, but I installed this version and when alarm is set (4
zones) Homekit now correctly displays armed night mode.

**Reply by K1LL3R234** [reply]
So it is working as indented?

**Comment by metaljay** [comment]
And the other texecom plugins don’t?

**Comment by 5xPa** [comment]
I certainly never got them to work

**Reply by 5xPa** [reply]
K1LLR234 Yes for example with other Texecom connect variants. To trigger automation on for example "armed night" I relied on a relay output connected to a "shelly uni" which changed state on alarm being armed.
Great job thanks

**Reply by K1LL3R234** [reply]
Thank you very much. I am happy it works.
